app/controllers/placements/support/organisations/users_controller.rb

```diff
@@ -26,8 +25,26 @@ def create
     end
   end
 
+  def remove; end
+
+  def destroy
+    membership = @user.memberships.find_by(organisation: @organisation)
+    if membership.present? && membership.destroy
+      RemoveUserService.call(@user, @organisation)
+      redirect_to_index
+      flash[:success] = t(".user_removed")
+    else
+      # TODO: There is no design for handling failure? Eg, if the membership has already been deleted?
+      redirect_to_index
+    end
```

Comment:
This is a bit of a weird edge case, isn't it. It's entirely possible that this could occur, but without knowing the reasons (e.g. a race condition, or a database connection issue, or a faulty flux capacitor..) it's difficult to know how to gracefully handle it without making assumptions.

I wonder if we would be better using `membership.destroy!` – in other words, keeping it simple. And then if/whenever this specific failure occurs, we'll have error tracking so we can inspect and debug it.

My hunch is that we'll never, or very rarely, see it happen in the real world. So it's probably not worthy of our attention until it actually affects users.